plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -417,72 +390,71 @@ public void testMemSqlCreatedParameterizedChar()
                 .execute(getQueryRunner(), memSqlCreateAndInsert("tpch.memsql_test_parameterized_char"));
     }
 
-    private DataTypeTest memSqlCharTypeTest()
+    private SqlDataTypeTest memSqlCharTypeTest()
     {
-        return DataTypeTest.create()
-                .addRoundTrip(charDataType("char", 1), "")
-                .addRoundTrip(charDataType("char", 1), "a")
-                .addRoundTrip(charDataType(1), "")
-                .addRoundTrip(charDataType(1), "a")
-                .addRoundTrip(charDataType(8), "abc")
-                .addRoundTrip(charDataType(8), "12345678")
-                .addRoundTrip(charDataType(255), "a".repeat(255));
+        return SqlDataTypeTest.create()
```

Comment:
Interesting that we aren't testing any chars with trailing whitespaces. i.e. value being inserted smaller than the data-type width. Let's add those (see PostgreSQL `testChar`).

Reply:
Existing tests already have "value being inserted smaller than the data-type width". e.g.
```java
                .addRoundTrip("char(5)", "'攻殻'", createCharType(5), "CAST('攻殻' AS char(5))")
```